RobotFramework_TestsuitesManagement/Config/CConfig.py

```diff
@@ -520,17 +520,25 @@ def __loadConfigFileLevel2(self):
             CConfig.sLoadedCfgError = str(error)
             logger.error(f"Loading of JSON configuration file failed! Reason: {CConfig.sLoadedCfgError}")
             raise Exception
-        
+        if self.sConfigName not in oSuiteConfig:
+            CConfig.sLoadedCfgError = f"Testsuite management - Loading configuration level 2 failed! \n \
+        The variant '{self.sConfigName}' is not defined in '{os.path.abspath(self.sTestSuiteCfg)}'"
```

Comment:
Hi Son,
in TML I display here a list of all possible variants.
It should be possible to get this from the json file which defines the variants.
Thank you,
 Thomas

Reply:
Hello Thomas,

I added the information of possible variants in the second commit of this pull request.

Thank you,
Son